modules/core/shared/src/main/scala/lucuma/core/model/Target.scala

```diff
@@ -273,14 +273,14 @@ trait TargetOptics { this: Target.type =>
     sidereal.andThen(SiderealTarget.epoch)
 
   /** @group Optics */
-  val properMotion: Optional[Target, ProperMotion] =
-    siderealTracking.andThen(SiderealTracking.properMotion.some)
+  val properMotion: Optional[Target, Option[ProperMotion]] =
```

Comment:
We discussed turning this into `Lens[Target, Option[ProperMotion]` but the structure won't allow it